docs/releases.rst

```diff
@@ -33,9 +33,9 @@ Upgrading
 Upgrade should run fine as long as you generate a new compose or stack
 configuration and upgrade your mailu.env.
 
-Please not that the shipped image for PostgreSQL database is deprecated.
+Please note that the shipped image for PostgreSQL database is deprecated.
 The shipped image for PostgreSQL is not maintained anymore from release 1.8.
-We recommend switching to an external database as soon as possible.
+We recommend switching to another database as soon as possible (SQLite or MySQL).
```

Comment:
```suggestion
We recommend switching to an external PostgreSQL database as soon as possible.
```

PostgreSQL will be supported in future, just the Mailu PostgreSQL image will be dropped. No need to switch to SQLite or MySQL.